docs/source/release_notes/release-notes.rst

```diff
@@ -32,6 +32,24 @@ Changelog
 .. <<<<------------------------------------------------------------------------------------------------------------->>>>
 
 =======
+develop
```

Comment:
0.27.0 was tagged a few days ago, but wasn't officially announced until last night due to testing hiccups with our large internal product.